### PR TITLE
#2/Update readme

### LINKED ISSUE #2: 🏷️ Basic template
Updates

- [ ] Add commitlint
- [ ] Change Name
- [ ] Add Readme

### PR DESCRIPTION
**Pull request by chekuda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
-# React boilerplate
+# :zap Template Navy React
```
Comment (oxmanroman) [inline README.md]: This should be
` # :zap: Template Navy React`

File: README.md [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+
+This template has been created for the use of the AMAZING [https://github.com/lightning-resume/lightning-resume]lightning-resume project
```
Comment (oxmanroman) [inline README.md]: I believe you wanted the links to show in a different way, please take a look [here](https://github.com/lightning-resume/template-navy-react/tree/2/Update-readme)

You should do something like: [my text](https://blabla.com \)

**Review by oxmanroman — CHANGES_REQUESTED** [review]

**Comment by 0xGorilla** [comment]
Could you please add a screenshot to the README?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
+# :zap: Template Navy React
 
-Make your resume simple and quick just importing the json generated by resume linkdin-pdf-parser
-with this React boilerplate
\ No newline at end of file
+This template has been created for the use of the AMAZING [lightning-resume project](https://github.com/lightning-resume/lightning-resume)
+
+##  Hot it works
+
+The template reads the JSON generated by [lightning-resume project](https://github.com/lightning-resume/lightning-resume).
```
Comment (0xGorilla) [inline README.md:7]:
```
The template reads the JSON generated by [linkedin-resume-parser](https://github.com/lightning-resume/linkedin-resume-parser).
```

**Review by oxmanroman — APPROVED** [review]

**Pull request merged**